lib/src/collection.dart

```diff
@@ -0,0 +1,40 @@
+// ignore_for_file: avoid_print
+import "dart:io";
+import "dart:async";
+import "package:burt_network/burt_network.dart";
+import "package:opencv_ffi/opencv_ffi.dart";
+import "udp.dart";
+import "constants.dart";
+import "camera.dart";
+
+
+
+
+class VideoCollection{
+  /// Holds a list of available cameras
+  Map<String, CameraManager> cameras = {};
```

Comment:
Maybe make this a `Map<CameraName, CameraManager>`? We're trying to make everything based on `CameraName`s

Reply:
Also, let's do the initialization here: 
```suggestion
  Map<String, CameraManager> cameras = {
    for (final name in CameraName.values) 
      name: CameraManager(
        camera: getCamera(name),
        details: defaultDetails.deepCopy(),
      );
  };
```

And then you can have at the global level: 
```dart
final defaultDetails = CameraDetails(resolutionWidth: 300, resolutionHeight: 300, quality: 50, fps: 24, status: CameraStatus.CAMERA_ENABLED);

Camera getCamera(CameraName name) => Platform.isWindows
  ? Camera.fromIndex(cameraIndexes[name])  // make [cameraIndexes] a `Map<CameraName, int>`
  : Camera.fromName(cameraNames[name]);